openapi/generated.json

```diff
@@ -320,13 +322,7 @@
             "$ref": "#/components/schemas/WebhookDetails"
           },
           "routing_algorithm": {
-            "$ref": "#/components/schemas/RoutingAlgorithm"
-          },
-          "custom_routing_rules": {
-            "type": "array",
-            "items": {
-              "$ref": "#/components/schemas/CustomRoutingRules"
-            }
+            "$ref": "#/components/schemas/serde_json.Value"
```

Comment:
Please update `value_type` for `routing_algorithm` in `CreateMerchantAccount` struct to be `Option<Object>`.